Settings

funayman opened the pull request:
removed about dialog; moved to a settings activity; added preferences (future proofing).

Xsixteen commented:
Code does not build due to:
```
  Fix the issues identified by lint, or add the following to your build script to proceed with errors:
  ...
  android {
      lintOptions {
          abortOnError false
      }
  }
  ...
  
  The first 3 errors (out of 5) were:
  /Users/Eric/AndroidStudioProjects/SuperGenki-OpenSource/super-genki-app/app/src/main/res/drawable/ic_sync_black_24dp.xml:8: Error: Use -0.25 instead of -.25 to avoid crashes on some devices [InvalidVectorPath]
              android:pathData="M12 4V1L8 5l4 4V6c3.31 0 6 2.69 6 6 0 1.01,-.25 1.97,-.7 2.8l1.46 1.46C19.54 15.03 20 13.57 20 12c0,-4.42,-3.58,-8,-8,-8zm0 14c-3.31 0,-6,-2.69,-6,-6 0,-1.01.25,-1.97.7,-2.8L5.24 7.74C4.46 8.97 4 10.43 4 12c0 4.42 3.58 8 8 8v3l4,-4,-4,-4v3z"/>
                                                                           ~~~~
  /Users/Eric/AndroidStudioProjects/SuperGenki-OpenSource/super-genki-app/app/src/main/res/drawable/ic_sync_black_24dp.xml:8: Error: Use -0.7 instead of -.7 to avoid crashes on some devices [InvalidVectorPath]
              android:pathData="M12 4V1L8 5l4 4V6c3.31 0 6 2.69 6 6 0 1.01,-.25 1.97,-.7 2.8l1.46 1.46C19.54 15.03 20 13.57 20 12c0,-4.42,-3.58,-8,-8,-8zm0 14c-3.31 0,-6,-2.69,-6,-6 0,-1.01.25,-1.97.7,-2.8L5.24 7.74C4.46 8.97 4 10.43 4 12c0 4.42 3.58 8 8 8v3l4,-4,-4,-4v3z"/>
                                                                                     ~~~
  /Users/Eric/AndroidStudioProjects/SuperGenki-OpenSource/super-genki-app/app/src/main/res/drawable/ic_sync_black_24dp.xml:8: Error: Use 0.25 instead of .25 to avoid crashes on some devices [InvalidVectorPath]
              android:pathData="M12 4V1L8 5l4 4V6c3.31 0 6 2.69 6 6 0 1.01,-.25 1.97,-.7 2.8l1.46 1.46C19.54 15.03 20 13.57 20 12c0,-4.42,-3.58,-8,-8,-8zm0 14c-3.31 0,-6,-2.69,-6,-6 0,-1.01.25,-1.97.7,-2.8L5.24 7.74C4.46 8.97 4 10.43 4 12c0 4.42 3.58 8 8 8v3l4,-4,-4,-4v3z"/>                                 
```

Additionally it looks like there is significant dummy code in the project.

Xsixteen commented on app/src/main/res/values/strings.xml:
```diff
@@ -42,4 +22,77 @@
+    <!-- Example General settings -->
+    <string name="pref_header_general">General</string>
+
+    <string name="pref_title_social_recommendations">Enable social recommendations</string>
```
I dont think any of this should role to prod if it isnt used yet.

funayman commented:
updated changes per your request.
I cannot reproduce the first error you mentioned. I pulled down a clean version and switched to the branch and it compiles w/o any errors.
